### ISSUE TITLE
size of g_pfnVectors is 0

### ISSUE THREAD

**Issue by metebalci** [STMicroelectronics/STM32CubeH5#10]
The size of g_pfnVectors in startup code is set at the wrong place and it is set to 0.

```
$ arm-none-eabi-objdump -t startup_stm32h563zitx.o
...
00000000 l    d  .isr_vector    00000000 .isr_vector
...
```

**Comment by TOUNSTM** [comment]
Hello @metebalci,

Thank you for this report. As this issue is a duplicate of [STM32CubeH5#11](https://github.com/STMicroelectronics/STM32CubeH5/pull/11) please allow me thus to close this one and keep the other.

With regards,

**Comment by ALABSTM** [comment]
Fixed in d9bc683963de1190859e5fa57d3dbfd2ae3c4fd2 (part 1)

**Comment by ALABSTM** [comment]
Fixed in ced6e5e11a9de5a54edbd0557d50bc98a711fe35 (part 2)
